plugins/modules/docker_container.py

```diff
@@ -52,6 +52,11 @@
       - List of capabilities to drop from the container.
     type: list
     elements: str
+  cgroup_parent:
+    description:
+      - Specify the parent cgroup for the container.
```

Comment:
I see in the tests you added alongside you pass an empty string, what does it do? perhaps it should be added this is supported and the behaviour ?

Reply:
Well, the test doesn't really do anything :) The problem with testing this properly in integration tests is that you need to know which cgroups are around on the systems you run the tests on, which is pretty much impossible since they can run on any system (if you run `ansible-test integration --docker xxx` locally, your own docker daemon will be used, and thus it depends on your local system). That's why I put in an empty string, which by the module is treated as "value specified by user", but by the Docker daemon as "no value supplied" resp. "user does not care whether/which cgroup parent to use". This test mainly makes sure that all related code is run through at least once.